### ISSUE TITLE
<si-date-range> loses meaning of start and end for screenreaders

### ISSUE THREAD

**Issue by FelixHirschel** [siemens/element#1376]
### Prerequisites

- [x] I have read the [Contributing Guidelines](https://github.com/siemens/element/blob/main/CONTRIBUTING.md).
- [x] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### Feature request

The date-range component's inputs should have aria-labels which detail if the user currently focuses the start or the end date.

cc: @Danisand 

### Use case

Currently, when users interact with the date-range component with a screen reader the following may happen:

1. They tab into the inputs and their placeholders are read out by the screenreader ("Start"/"End")
2. They enter any value
3. They tab to the next input
4. They tab back
5. The specific date is read out but, not the placeholder. 
 
It is not clear for the user, if they are now in the first or second date
